stable/index.yaml

```diff
@@ -3,7 +3,7 @@ entries:
   argo-cd:
   - apiVersion: v1
     appVersion: 1.2.0
-    created: "2020-07-15T19:10:59.142218724Z"
+    created: "2020-07-15T18:33:35.674604774-05:00"
```

Comment:
that's odd, should it have updated all these non-Kommander charts?

Reply:
helm rebuilds the whole index it seems.

Reply:
the previous timestamps all are within milliseconds, so it seems like this happens for every build.

Reply:
yeah guessing it probably happens from CI or every time someone pushes up a new chart

Reply:
yup, if you look at any of the commits, dispatch bot does the same thing https://github.com/mesosphere/charts/commits/gh-pages